README.md

```diff
@@ -21,6 +21,7 @@
   - [Collect a customer's card PIN for a payment and defer the capture of the payment to the server](#collect-a-customers-card-pin-for-a-payment-and-defer-the-capture-of-the-payment-to-the-server)
   - [Capture a payment immediately](#capture-a-payment-immediately)
   - [Refund a payment](#refund-a-payment-pos-terminal-only) **(POS-only)**
+  - [Collect a customer's card PIN for a refund and defer the capture of the refund to the server](#collect-a-customers-card-pin-for-a-refund-and-defer-the-capture-of-the-refund-to-the-server) **(POS-only)**
```

Comment:
```suggestion
  - [Collect a customer's card PIN for a refund and defer the completion of the refund to the server](#collect-a-customers-card-pin-for-a-refund-and-defer-the-capture-of-the-refund-to-the-server) **(POS-only)**
```

Is "capturing a refund" accurate terminology here? What are your thoughts on saying "completion" instead?

Reply:
I went with that language because I vaguely remembered we wanted to avoid "processing" in a previous PR, but I couldn't remember why. 

I have no strong feelings and will edit to this so we can ship ASAP.